### ISSUE TITLE
Can't get fabric8 Vagrant box to work on Fedora 23

### ISSUE THREAD

**Issue by bkabrda** [fabric8io/fabric8-installer#39]
Hi,
I'm trying to setup fabric8 Vagrant box on Fedora 23 using approach from [1].
- I installed VirtualBox from upstream (VirtualBox-5.0-5.0.10_104061_fedora22-1.x86_64) and Vagrant from official Fedora repos (vagrant-1.7.4-2.fc23.noarch).
- I installed the landrush plugin (0.18.0).
- I wanted to set up the dnsmasq rules, but dnsmasq conflicts with NetworkManager, so I skipped this step; I'm assuming that this step isn't really necessary and I could just access 172.28.128.4 instead of vagrant.f8 in my browser. (If this assumption isn't valid, I'd appreciate a how-to on setting things up with NetworkManager - I can't really turn it on, since then my network would stop working :).)
- I checked out this repo and did "vagrant up --provider virtualbox" in `vagrant/openshift` directory.
- After a while, the installation seemed to had finished successfully, so I tried to access 172.28.128.4. I got (for both http and https):

```
503 Service Unavailable
No server available to handle the request. 
```

I tried to access the OpenShift console (https://172.28.128.4:8443) and everything seems to be working fine; moreover, it seems that all the installed services are running allright, OpenShift is reporting no errors.

I'd really appreciate if someone could help me analyze this problem and get fabric8 running on my box. Please let me know what information you need and I'll try to provide it ASAP. Alternatively, I'd be open to doing an IRC session to speed things up.

Thanks!

[1] http://fabric8.io/guide/getStarted/vagrant.html

**Comment by jstrachan** [comment]
Note that there's typically nothing on http://172.28.128.4 or https://172.28.128.4 to see. 

The fabric8 console is here by default: http://fabric8.vagrant.f8/ does that work?

of you type: 

```
oc get route
```

you should see all the externally available services you can try from your web browser / curl. I wonder what that shows?

**Comment by bkabrda** [comment]
Ok, so here's my progress: I managed to find out that it's possible to use dnsmasq with NetworkManager by adding `dns=dnsmasq` to `/etc/NetworkManager/NetworkManager.conf` `main` section, then I created `/etc/NetworkManager/dnsmasq.d/vagrant-landrush.conf` containing `server=/vagrant.f8/127.0.0.1#10053` (as instructed at [1]).
When I restart NetworkManager, dnsmasq is started and I see this in my system journal:

```
Dec 01 11:36:09 dhcp-2-157.brq.redhat.com dnsmasq[26929]: cleared cache
Dec 01 11:36:09 dhcp-2-157.brq.redhat.com dnsmasq[26929]: using nameserver 127.0.0.1#10053 for domain vagrant.f8
```

So everything looks to be ok. I still _can't_ access http://fabric8.vagrant.f8, though:

```
[bkabrda@dhcp-2-157 openshift]$ curl fabric8.vagrant.f8
curl: (6) Could not resolve host: fabric8.vagrant.f8
```

`oc get route` inside the Vagrant box returns this:

```
NAME      HOST/PORT            PATH      SERVICE   LABELS             INSECURE POLICY   TLS TERMINATION
fabric8   fabric8.vagrant.f8             fabric8   provider=fabric8
```

[1] http://fabric8.io/guide/getStarted/vagrant.html

**Comment by sebastienblanc** [comment]
@bkabrda Hi ! I was wondering if you solved your issue since I'm facing the same (on F23) ?

**Comment by bkabrda** [comment]
I didn't, but would still be curious what the solution is.

**Comment by cmoulliard** [comment]
The workaround is to add manually the addresses within /etc/hosts file.

**Comment by sebastienblanc** [comment]
@cmoulliard yes that is what I did, in fact I think I have another issue because now I have a "connection refused"
